composer.json

```
@@ -29,5 +29,10 @@
     "psr-4": {
       "Spiral\\Nyholm\\Tests\\": "tests/"
     }
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-master": "1.2.x-dev"
+    }
```

```suggestion
  }
```

I think it can be eliminated